### DISCUSSION TITLE
Bigscreen Beyond support on Linux (Probably non-conformant DSC implementation on Linux)

### DISCUSSION THREAD

**Discussion by Hisanatos** [NVIDIA/open-gpu-kernel-modules#679] [General]
As per https://github.com/NVIDIA/open-gpu-kernel-modules/discussions/238#discussioncomment-6236333

I'm starting a new thread discussion Bigscreen Beyond problems on Linux on Nvidia. Thread was already oppened on https://forums.developer.nvidia.com/t/bigscreen-beyond-hmd-unable-to-be-initialized-on-nvidia-possibly-due-to-dsc-bpp-issue/294108/7 but we got no resposne from any Nvidia rep about if the issue is looked into, or will be looked in the future.

What I want to know is only if Nvidia is looking into it at all. There are people that are willing to suply debugging information about it or what doesn't work, so it works on AMD software after patches to MESA so it looks that the issue is not in Linux Kernel, nor on SteamVR side, but on GPU driver side.

I'd be grateful for any information.

**Comment by aritger** [comment]
I don't believe that problem is specific to the open-gpu-kernel-modules.  We shouldn't really use the github discussion infrastructure here for general problems with the NVIDIA Linux GPU driver.  Let's please keep that discussion on https://forums.developer.nvidia.com.  I'll try to reply there.

**Reply by Hisanatos** [reply]
Then I'm sorry, I didn't knew this was a repo for Open-Source driver problmes only. I've read the guidelines but I haven't got such an impression. But thanks for taking interest in the issue, this is all I wanted to know :bow:
